src/encoded/types/file.py

```diff
@@ -1,15 +1,44 @@
-from typing import Any, Dict, List, Optional, Union
-
+from typing import Any, Dict, Optional, Union
+from pyramid.view import view_config
+from pyramid.request import Request
 from encoded_core.types.file import (
     HREF_SCHEMA,
     UNMAPPED_OBJECT_SCHEMA,
     UPLOAD_KEY_SCHEMA,
     File as CoreFile,
 )
-from pyramid.request import Request
-from snovault import abstract_collection, calculated_property, load_schema
+from encoded_core.file_views import (
+    validate_file_filename,
+    validate_extra_file_format,
+    drs as CoreDRS,
+    download as CoreDownload,
+    post_upload as CorePostUpload,
+    get_upload as CoreGetUpload
```

Comment:
Consider adding tests for these to ensure working as desired and aren't unintentionally broken in future.